scripts/Interface/ui/reflectometer/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+
+# List of UIs to Auto convert
+set( UI_FILES
+refl_columns.ui   
+data_refl_simple.ui
```

Comment:
I think data_refl_simple.ui can be removed - it looks like this was part of the ORNL stuff